README.md

```diff
@@ -82,7 +82,130 @@ Create an `examples` object on each parameter that needs to go into a group in t
   }
 }
 ```
-`loast` will go through and execute a test against the endpoint for each grouping it finds, including any required parameters in each request. 
+`loast` will go through and execute a test against the endpoint for each grouping it finds, including any required parameters in each request.
+
+## Parameter Groups
```

Comment:
I think it may be useful to provide a sample path object that uses the `examples` field for some parameters and the `example` field for others and shows the resulting example groups

Reply:
done